f2/src/forms/ConfirmationForm.js

```diff
@@ -3,21 +3,52 @@ import PropTypes from 'prop-types'
 import React from 'react'
 import { Trans } from '@lingui/macro'
 import { jsx } from '@emotion/core'
-import { Form } from 'react-final-form'
+import { Form, Field } from 'react-final-form'
 import { NextAndCancelButtons } from '../components/next-and-cancel-buttons'
-import { Box } from '@chakra-ui/core'
+import { Stack, Box, FormControl } from '@chakra-ui/core'
+import { FormLabel} from '../components/FormLabel'
+import { TextArea } from '../components/text-area'
+import { FormHelperText } from '../components/FormHelperText'
 
 export const ConfirmationForm = props => {
   return (
     <React.Fragment>
       <Form
         onSubmit={props.onSubmit}
         render={({ handleSubmit }) => (
+
+          <Stack as="form" onSubmit={handleSubmit} shouldWrapChildren spacing={6}>
+
+          <Field name="tellUsMore">
+            {props => (
+
+              <FormControl>
+                <FormLabel>
+                  <Trans id="confirmationPage.tellUsMore"/>
+                </FormLabel>
+
+                <FormHelperText>
+                  <Trans id="confirmationPage.tellUsMoreHelperText"/>
+                </FormHelperText>
+                
+                <TextArea id= "tellUsMore"
```

Comment:
I dont know if this will break anything, but there is a space between the = and "